### PR TITLE
fix(24.04): fix release errors

### PR DESCRIPTION
**Pull request by cjdcordeiro**
This release is in development and thus packages keep changing upstream.

This PR fixes the issues found in the [current tests](https://github.com/canonical/chisel-releases/actions/runs/8019314799). Those include various path re-definitions and the removal of `libunwind-13`

### CODE REVIEW

**Review by rebornplusplus — APPROVED** [review]
Wow, that is a lot of changes. Thanks for adding the PR!

File: slices/base-files.yaml [hunk 1]
```diff
@@ -16,14 +16,14 @@ slices:
 
   bin:
     contents:
-      /bin/:
-      /sbin/:
```
Comment (letFunny) [inline slices/base-files.yaml:20]: Out of curiosity, why are we removing the trailing slash here? It seems like we are using it below, e.g. `/usr/bin/:`

Comment (cjdcordeiro) [reply slices/base-files.yaml:20]: these are symlinks now

**Review by linostar — APPROVED** [review]
LGTM, thanks!

File: slices/base-files.yaml (hunk 1 above)
Comment (letFunny) [reply slices/base-files.yaml:20]: That means that it is extracting symlinks included in the tarball?

**Review by letFunny — APPROVED** [review]
Thanks!

File: slices/base-files.yaml (hunk 1 above)
Comment (cjdcordeiro) [reply slices/base-files.yaml:20]: Yes, exactly

Comment (cjdcordeiro) [reply slices/base-files.yaml:20]: Please re-open if you think this should be done differently :)

**Pull request merged**